src/wgxpath.js

```diff
@@ -221,16 +222,17 @@ wgxpath.XPathNSResolver_ = function(node) {
 
 
 /**
- * Installs the library. This is a noop if native XPath is available.
+ * Installs the library. This is a noop if native XPath is available or the 2nd parameter is true.
```

Comment:
@Dominator008 Yes, it should be. Pushed up changes to documentation. Thanks, good catch.